sql/_35_fig_cake/plcsql/basic/cases/test_serial.sql

```diff
@@ -0,0 +1,20 @@
+DROP SERIAL IF EXISTS demo_pl_serial;
+CREATE SERIAL demo_pl_serial;
+
+create or replace function demo_global_semantics_serial() return numeric as
+i numeric;
+begin
+    i := demo_pl_serial.NEXT_VALUE;
+    return i;
+end;
+
+CAll demo_global_semantics_serial ();
+CAll demo_global_semantics_serial ();
+CAll demo_global_semantics_serial ();
+CAll demo_global_semantics_serial ();
+
```

Comment:
select demo_pl_serial.CURRVAL [from dual] ;
or
select current_val from db_serial where name = 'demo_pl_serial' ;
or
select demo_global_semantics_serial ();

would be need to check  created function works well.